src/hooks/useIssuesData.tsx

```diff
@@ -1,12 +1,12 @@
-import { Node } from "typescript";
 import { useIssuesContext } from "../contexts/IssuesContext";
 
 export const useIssuesData = () => {
-  const { setIssuesData } = useIssuesContext();
-  const fetchIssues = (inputValue: string) => {
+  const { setIssuesData, setSpinner } = useIssuesContext();
+  const fetchIssues = async (inputValue: string) => {
+    setSpinner(true);
     const githubAccountName: string = inputValue.split("/")[3];
     const githubRepositoryName: string = inputValue.split("/")[4];
-    fetch(
+    await fetch(
```

Comment:
`async await` and `then` flows are the same and shouldn't be used simultaneously, choose one